akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -140,7 +140,7 @@ object ActorSystem {
   def apply(name: String, config: Option[Config] = None, classLoader: Option[ClassLoader] = None, defaultExecutionContext: Option[ExecutionContext] = None): ActorSystem = {
     val cl = classLoader.getOrElse(findClassLoader())
     val appConfig = config.getOrElse(ConfigFactory.load(cl))
-    new ActorSystemImpl(name, appConfig, cl, defaultExecutionContext).start()
+    new ActorSystemImpl(name, appConfig, cl, defaultExecutionContext, None).start()
```

Comment:
right, there are other things that are interesting from the untyped system, but perhaps you have exposed them explicitly in `typed.ActorSystem`. I'll review the change.